extensions/wikia/ContentReview/controllers/ContentReviewApiController.class.php

```diff
@@ -207,13 +208,55 @@ public function showTestModeNotification() {
 		$notification = wfMessage( 'content-review-test-mode-enabled' )->escaped();
 		$notification.= Xml::element(
 			'a',
-			[ 'class' => 'content-review-test-mode-disable', 'href' => '#' ],
-			wfMessage( 'content-review-test-mode-disable' )->plain()
+			[ 'class' => 'content-review-module-test-mode-disable', 'href' => '#' ],
+			wfMessage( 'content-review-module-test-mode-disable' )->plain()
 		);
 
 		$this->notification = $notification;
 	}
 
+	/**
+	 * Prepares parts for rendering status module
+	 * Returns data required by ContentReviewModule.mustache
+	 * @throws MWException
+	 */
+	public function renderStatusModal() {
+		global $wgCityId;
+		$pageName = $this->request->getVal( 'pageName' );
+		$title = Title::newFromText( $pageName );
+
+		/* Get page status */
+		$pageStatus = \F::app()->sendRequest(
+			'ContentReviewApiController',
+			'getPageStatus',
+			[
+				'wikiId' => $wgCityId,
+				'pageId' => $title->getArticleID(),
+			],
+			true
+		)->getData();
+
+		/* Render status module */
+		$res = \F::app()->sendRequest(
+			'ContentReviewModule',
+			'executeRender',
+			[
+				'pageStatus' => $pageStatus,
+				'latestRevisionId' => $title->getLatestRevID(),
+			],
+			true
+		)->getData();
+
+		/* Add link to help page to result */
+		$helpTitle = Title::newFromText( wfMessage( 'content-review-module-help-article' )->escaped() );
```

Comment:
The link it constructed in modal template
https://github.com/Wikia/app/blob/dev/resources/wikia/ui_components/modal/templates/modal_default.mustache#L17-17